docs/application-author.md

```diff
@@ -0,0 +1,344 @@
+# Application Workload Author's Guide
+
+
+
+- [Introduction to Service Binding](#Introduction-to-Service-binding)
+
+- [Backing Service providing binding metadata](#Backing-service-providing-binding-metadata)
+
+- [Backing Service not providing binding metadata](#Backing-service-not-providing-binding-metadata)
+  * [Annotate Service Resources](##Annotate-service-resources)
+  * [Detect Binding Resources](#Detect-binding-resources)
+  * [Compose custom binding variables](#Compose-custom-binding-variables)
+
+- [Accessing the binding data from the application](#Accessing-the-binding-data-from-the-application)
+
+- [Binding non-podSpec-based application workloads]([#Binding-non-podSpec-based-application-workloads])
+
+<!-- toc -->
+
+
+# Introduction to Service Binding
+
+A Service Binding involves connecting an application to one or more backing services using a binding secret generated for the purpose of storing information to be consumed by the application.
+
+``` yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+  name: binding-request
+  namespace: service-binding-demo
+
+spec:
+  ## Workload where binding information is
+  ## injected.
+
+  application:
+    name: java-app
+    group: apps
+    version: v1
+    resource: deployments
+
+  ## One or more Service resources from which
+  ## binding information is collected
+  services:
+  - group: database.example.com
+    version: v1alpha1
+    kind: DBInstance
+    name: db
+
+  - group: database.example.com
+    version: v1alpha1
+    kind: DBCredentials
+    name: db
+
+  - group: "route.openshift.io"
+    version: v1
+    kind: Route
+    name: auth-service
+```
+
+The application workload expects binding metadata to be present on the Kubernetes Resources representing the backing service.
+
+As shown above, you may also directly use a `Configmap` or a `Secret` itsef as a service resource that would be used as a source of binding information.
```

Comment:
nit- ConfigMap

Reply:
https://github.com/redhat-developer/service-binding-operator/pull/707/files#diff-60da86c468a457b0c38c7e96ee96ae29R9

and following sentences